docs/src/pages/demos/badges/badges.md

```diff
@@ -33,12 +33,11 @@ The badge auto hides with badgeContent is zero. You can override this with the `
 
 {{"demo": "pages/demos/badges/BadgeVisibility.js"}}
 
-## Customized Badge
+## Customized badge
```

Comment:
@mbrookes The capitalization of the documentation headers is inconsistent. Do you agree with this format?

Reply:
I guess it depends whether badge refers to the UI element (a badge) or the React component `Badge`. :)

Reply:
I think that it's better to refer to the UI element. It might be composed a multiple React components.

Reply:
In which case, 
```suggestion
## Customized badges
```
reads better.

Reply:
👍